lib/crunch-cmd.ML

```diff
@@ -143,11 +145,12 @@ end
 signature CRUNCH =
 sig
     type extra;
-    (* Crunch configuration: theory, naming scheme, name space, wp rules, wps_rules,
-    constants to ignore, simp rules, constants to not ignore, unfold rules*)
-    type crunch_cfg = {ctxt: local_theory, prp_name: string, nmspce: string option,
-        wp_rules: (string * thm) list, wps_rules: thm list, igs: string list, simps: thm list,
-        ig_dels: string list, rules: thm list};
+    (* Crunch configuration: theory, property name, name space, wp rules, wps_rules,
+       constants to ignore, simp rules, constants to not ignore, unfold rules *)
+    type crunch_cfg =
+      {ctxt: local_theory, prp_name: string, nmspce: string option,
+       wp_rules: (string * thm) list, wps_rules: thm list, igs: string list,
+       simps: thm list, ig_dels: string list, rules: thm list};
```

Comment:
stared at this for a bit... it's only re-styling, right?

Reply:
Yeah, it's just a re-styling now.

My original implementation of this option changed the type, and when I undid the rest of those changes I decided that this style is more consistent with the rest of the file. Apologies for any time you spent staring at it.